partcad/src/partcad/__init__.py

```diff
@@ -60,18 +69,20 @@
 
 # TODO(clairbee): move the below to `logging_sentry.py`
 if not sentry_sdk.is_initialized() and user_config.get_string("sentry.dsn"):
-    critical_to_ignore = [
+    critical_to_ignore = (
         "action_start: ",
         "action_end: ",
-    ]
-    debug_to_ignore = [
+        "process_start: ",
+        "process_end: ",
+    )
+    debug_to_ignore = (
         "Starting action",
         "Finished action",
-    ]
+    )
 
     def before_send(event, hint):
         # Reduce noise in logs (drop events from "with logging.Process():")
-        if event.get("level") == "critical":
+        if event.get("level") in ["critical", "fatal"]:
             # from logging_ansi_terminal.py
             message = event.get("logentry", {}).get("message")
             if message and message.startswith(critical_to_ignore):
```

Comment:
This one didn't work as expected, `critical_to_ignore` had to be a tuple and not a list. This is fix for:

- https://discord.com/channels/1308854595987968051/1329317294609928202/1329381235054739508

![image](https://github.com/user-attachments/assets/5beb08f2-ef08-4cc3-9a96-9f6f0a8f51c0)